src/EventEdition/RepeatPanel.vala

```diff
@@ -17,7 +17,7 @@
  * Authored by: Jaap Broekhuizen
  */
 
-public class Maya.View.EventEdition.RepeatPanel : Gtk.Grid {
+ public class Maya.View.EventEdition.RepeatPanel : Gtk.Grid {
```

Comment:
```suggestion
public class Maya.View.EventEdition.RepeatPanel : Gtk.Grid {
```
looks like an oops :slightly_smiling_face:

Reply:
I don't know how this appeared :) Thanks!